reactor-core/src/test/java/reactor/core/publisher/TracesTest.java

```diff
@@ -176,24 +176,8 @@ public void shouldSanitizeTrue() {
 		assertThat(Traces.shouldSanitize("reactor.core.publisher.Flux.onAssembly")).isTrue();
 		assertThat(Traces.shouldSanitize("reactor.core.publisher.ParallelFlux.onAssembly")).isTrue();
 		assertThat(Traces.shouldSanitize("reactor.core.publisher.SignalLogger")).isTrue();
-		assertThat(Traces.shouldSanitize("FluxOnAssembly.")).isTrue();
-		assertThat(Traces.shouldSanitize("MonoOnAssembly.")).isTrue();
-		assertThat(Traces.shouldSanitize("MonoCallableOnAssembly.")).isTrue();
-		assertThat(Traces.shouldSanitize("FluxCallableOnAssembly.")).isTrue();
-		assertThat(Traces.shouldSanitize("OnOperatorDebug")).isTrue();
 		assertThat(Traces.shouldSanitize("reactor.core.publisher.Hooks")).isTrue();
-		assertThat(Traces.shouldSanitize(".junit.runner")).isTrue();
```

Comment:
@smaldini do you remember why all these cases where sanitized?
@bsideup can you explain why you removed these?

Reply:
these string operations add overhead and I was trying to minimize it. 
The reason why I safely removed lines like this one because they always come after user's code and current implementation stops at it. But I'm happy that you raised the question, I would be happy to learn if there are some use cases where this is not true 👍

Reply:
By user code you mean first call site of a `reactor.core.publisher` class?
The old implementation would never stop, but if you can confirm the two example below would be traced, then this should be fine.

```java
public Flux<String> myMethodOne() {
    return myMethod(1);
}

public Flux<String> myMethod(int param) {
    return Flux.range(0, param)
        .map(i -> "" + i);
}
```

Calling the above method with param == 1 would result in 2 flux method calls (`Flux.range(x, 1)` being an alias of `Flux.just(x)`), so the traceback should at minimum show something like:

```
    Flux.java(1185) - just(T)
    Flux.java(1627) - range(int, int)
    com.acme.MyClass(123) - myMethod(int)
```
but preferably:
```
    Flux.java(1185) - just(T)
    Flux.java(1627) - range(int, int)
    com.acme.MyClass(123) - myMethod(int)
    com.acme.MyClass(119) - myMethodOne()
```

Reply:
I think the old implementation didn't handle it either? Otherwise, what if it went thru 10s of user methods? We show single `Operator -> Code location` entries, not the stacktrace, actually :)

Reply:
"user code" = something not from Reactor, e.g. `com.acme.MyClass`, or `some.cool.framework.Whatever` (if it called the operator)